src/wesnoth.cpp

```diff
@@ -941,7 +941,7 @@ static void restart_process(const std::vector<std::string>& commandline)
 #define error_exit(res) return res
 #endif
 
-#ifdef __APPLE__
+#ifdef __IPHONEOS__
```

Comment:
The mainline Mac build doesn't really use dependency management... the expectation is that you dump the dependencies into folders ("lib" and "Headers") within the project folder. (I had nothing to do with this, for the record.)